docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -575,7 +575,7 @@ It’s also possible to use so-called builtin variables in addition to user-defi
 - `cluster.topology.{version,class}`
 - `cluster.topology.controlPlane.{labels,annotations,replicas,version}`
     - **Note**: these variables are only available when patching control plane or control plane machine templates.
-- `cluster.topology.machineDeployment.current.{labels,annotations,replicas,version,class}`
+- `cluster.topology.machineDeployment.current.{labels,annotations,replicas,version,class,name,topologyName}`
```

Comment:
My plan was to drop it in a separate PR depending on the feedback (or lack thereof) on https://github.com/kubernetes-sigs/cluster-api/issues/5577. Mainly to avoid controversial discussions about it on this PR.